### PR TITLE
added detailed report and csv support for httptrace fields

### PR DESCRIPTION
**Pull request by JRonak**

### CODE REVIEW

File: requester/print.go [hunk 1]
```diff
@@ -99,54 +107,68 @@ func (r *report) finalize() {
...
 		r.printLatencies()
-	}
+		if len(r.errorDist) > 0 {
+			r.printErrors()
+		}
 
-	if len(r.errorDist) > 0 {
-		r.printErrors()
+		if r.trace {
+			fmt.Printf("\nDetailed Report:\n")
```
Comment (rakyll) [inline requester/print.go]: this is too much info. i am not sure if we want to show a breakdown for all latencies.

Comment (JRonak) [inline requester/print.go]: We can avoid displaying histogram, instead display the average, fastest and slowest?

Comment (rakyll) [inline requester/print.go]: Avoiding to display histogram sounds good.

Comment (JRonak) [inline requester/print.go]: Done, thanks!

**Comment by rakyll** [comment]
Thankss!!

**Pull request merged**